en/payloads/README.md

```diff
@@ -25,27 +25,41 @@ The following topics show how to *connect* your camera configure PX4:
 Cargo drones commonly use servos/actuators to trigger cargo release, control winches, etc.
 PX4 supports servo and GPIO triggering via both RC and MAVLink commands.
 
-### Example Mission (in QGC)
-
-Use the [MAV_CMD_DO_SET_ACTUATOR](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ACTUATOR) MAVLink command to trigger one of the payload actuators.
-
 ### RC Triggering
 
 You can map up to three RC channels to control servos/actuators attached to the flight controller using the parameters [RC_MAP_AUX1](../advanced_config/parameter_reference.md#RC_MAP_AUX1) to [RC_MAP_AUX3](../advanced_config/parameter_reference.md#RC_MAP_AUX3).
 
-The RC channels are then *usually* mapped to the `AUX1`, `AUX2`, `AUX3` outputs of your flight controller (but they don't have to be).
+The RC channels are *usually* mapped to the `AUX1`, `AUX2`, `AUX3` outputs of your flight controller - _but they don't have to be_.
 You can check which outputs are used for RC AUX passthrough on your vehicle in the [Airframe Reference](../airframes/airframe_reference.html).
-For example, [Quadrotor-X](../airframes/airframe_reference.html#quadrotor-x) has the normal mapping: "**AUX1:** feed-through of RC AUX1 channel".
+For example, [Quadrotor-X](../airframes/airframe_reference.md#quadrotor-x) has the normal mapping: "**AUX1:** feed-through of RC AUX1 channel", "**AUX2:** feed-through of RC AUX2 channel", "**AUX3:** feed-through of RC AUX3 channel".
 
-If your vehicle has no mapping then you can add one by using a custom [Mixer File](https://docs.px4.io/master/en/concept/mixing.html) that maps [Control group 3](../concept/mixing.md#control-group-3-manual-passthrough) outputs 5-7 to your desired port(s).
+If your vehicle doesn't specify RC AUX feed-through outputs, then you can add them using using a custom [Mixer File](../concept/mixing.md) that maps [Control group 3](../concept/mixing.md#control-group-3-manual-passthrough) outputs 5-7 to your desired port(s).
 An example of such a mixer is the default passthrough mixer: [pass.aux.mix](https://github.com/PX4/PX4-Autopilot/blob/master/ROMFS/px4fmu_common/mixers/pass.aux.mix).
 
+:::note
+The same outputs used for "feed-through of RC AUX" may also be set using a MAVLink command (see [below](#mission-triggering)).
+PX4 will use the last value set through either mechanism.
+:::
+
+
+### Mission Triggering
+
+You can use the [MAV_CMD_DO_SET_ACTUATOR](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_ACTUATOR) MAVLink command to set (up to) three actuators values at a time, either in a mission or as a command.
+
+Command parameters `param1`, `param2`, and `param3`, are mapped to the _same outputs_ as are used for [RC triggering](#rc-triggering).
+Usually these are the `AUX1`, `AUX2`, `AUX3` outputs of your flight controller (the RC section above explains how to check).
+The other command parameters (`param4` to `param7`) are unused/ignored by PX4.
+
+The parameters take normalised values between -1 and 1.
```

Comment:
@bkueng "How" are values normalised, and does it matter to the end user/integrator. 

What do I mean? Well an end user will get a setup vehicle and might be told that the particular actuator they are using can be controlled between a resting position of "0" and a full range of "1" and a full negative range of "-1". Presumably they might also be told that the actuator has just a range from 0 to 1, and the setup needs to ensure that negative values are pinned to 0.

So an integrator needs to understand what they have to do in order to make that mapping happen. I am assuming perhaps that the values are just normalised between PWM_AUX_MINx and PWM_AUX_MAXx but I'm not sure then how you'd support the "0 to 1" case. Also what other servo values should be set.

Does that make sense?

Reply:
@bkueng Thanks. I have updated the docs to say this and I will merge

But I think it is a little awkward. In the case where you have an actuator that moves from X to Y, you have to specify values between -1 and 1, while 0 and 1 would be much more natural. I suspect the design of the -1 range was for cases that  have a "center position, that can go positive or negative.